libs/utils/utils.js

```diff
@@ -138,6 +138,7 @@ export const MILO_EVENTS = { DEFERRED: 'milo:deferred' };
 
 const LANGSTORE = 'langstore';
 const PAGE_URL = new URL(window.location.href);
+const SLD = PAGE_URL.hostname.includes('.aem.') ? 'aem' : 'hlx';
```

Comment:
We wanted to export the const to re-use it in other files, right?
Since we don't use it right now, I'd add that later on though

Reply:
I remember we discussed it, but I think the conclusion for all other files was: you either port your whole project or you don't. So it's one or the other, we shouldn't have a need to detect it outside utils, where we can't be sure whether the consumer has migrated or not.

Reply:
And then in the core blocks where needed, we'd also just have the check for `new URL(window.location.href).hostname.includes('.aem.') ? 'aem' : 'hlx';` ?

So we can just check for the whole pattern after consumers migrated and remove the code again, rather than caring about other projects having imported that variable?